x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/tool/GenerateElasticPasswordHash.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.security.tool;
```

Comment:
Given its scope, it probably sits better in the enrollment tool namespace (I see the test class does)